### PR TITLE
Add spectralib

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/spectralib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  fn: spectralib-{{ version }}.tar.gz
+  url: https://github.com/yixuan/spectra/archive/v{{ version }}.tar.gz
+  sha256: 339ae9221309a128b8d937ca59b77d9b30aeceacb4ef2d2df13f6f7cde7fa3f3
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```
Comment (xhochy) [inline recipes/spectralib/meta.yaml]: Do you really need compilers? The script looks quite simple.

Comment (jschueller) [reply recipes/spectralib/meta.yaml]: no, I've tried this as the windows build keeps failing

Comment (jschueller) [reply recipes/spectralib/meta.yaml]: @xhochy I'm disabling windows for now, I'll re-enable it in the feedstock

**Comment by jschueller** [comment]
thanks @xhochy

**Pull request merged**